build.go

```diff
@@ -97,14 +97,41 @@ func Build(entryResolver EntryResolver, dependencyService DependencyService, pro
 
 		var launchMetadata packit.LaunchMetadata
 		if launch {
+			command := fmt.Sprintf(`nginx -p $PWD -c "%s"`, nginxConfPath)
 			launchMetadata.Processes = []packit.Process{
 				{
 					Type:    "web",
-					Command: fmt.Sprintf(`nginx -p $PWD -c "%s"`, nginxConfPath),
+					Command: command,
 					Default: true,
 				},
 			}
 			launchMetadata.BOM = bom
+
+			shouldReload, err := checkLiveReloadEnabled()
+			if err != nil {
+				return packit.BuildResult{}, err
+			}
+
+			if shouldReload {
+				launchMetadata.Processes = []packit.Process{
+					{
+						Type:    "web",
+						Command: "watchexec",
+						Args: []string{
+							"--restart",
+							"--watch", context.WorkingDir,
+							command,
+						},
+						Default: true,
+						Direct:  true,
+					},
+					{
+						Type:    "no-reload",
+						Command: command,
+						Direct:  true,
```

Comment:
The process won't be direct if its a single string like we have above. To run a direct process, you'd need to break the command up into an executable, `nginx`, and the arguments, `[]string{"-p", "${PWD}", "-c", nginxConfPath}`. Still, it won't be possible to run this directly because there is an environment variable in the arguments that needs to be run through a shell to be evaluated.

I think we should just remove the `Direct: true` line here so that this runs using a shell for the time being.